decidim-generators/lib/decidim/generators/app_generator.rb

```diff
@@ -152,6 +156,10 @@ def authorization_handler
         end
       end
 
+      def eager_load
+        gsub_file "config/environments/test.rb", "config.eager_load = false", "config.eager_load = true" if options[:ci]
```

Comment:
I guess we can check `ENV["CI"]` here directly and it's not necessary to add one more option to the generator just for this?

Reply:
I felt a bit weirder to me for some reason, as `tasks` have more to do with `system` and the generator is more of a standalone tool. Do you really feel it's better this way?

Reply:
Yeah, I think it unnecessarily complicates the generator with an option that nobody will use, only our CI. But it's not a very strong opinion so I'm good with keeping it!

Reply:
Also, it's maybe more explicit to check this inside the `test.rb` file, otherwise we're forcing the generated application to always eager load in testing, which is something we don't consider good ourselves (since we do it only in CI).

So maybe something like

```ruby
gsub_file "config/environments/test.rb", "config.eager_load = false", "config.eager_load = !ENV["CI"].nil?"
```
?

Reply:
Oh, I see now that you only pass this `--ci` flag in the `test_app` task, and only if CI is on. So just ignore my previous message since what you're doing is correct :flushed:.

Reply:
@josepjaume I gave this a second though and I think you're right :+1:

Reply:
Hahaha, I was going to ask that first time I reviewed this, but CI was not done yet... :)

Not sure if we should do this, but using eager_load in CI is definitely an interesting idea!